src/pages/{StoreProduct.slug}/p.tsx

```diff
@@ -96,7 +96,7 @@ function Page(props: Props) {
         Sections: Components imported from '../components/sections' only.
         Do not import or render components from any other folder in here.
       */}
-      <h1 className="temp-offscreen">{title}</h1>
+      <h1>{title}</h1>
```

Comment:
This is causing the following on PDPs:
![CleanShot 2022-01-12 at 20 05 13](https://user-images.githubusercontent.com/381395/149237315-f5ca8327-238d-40dc-ba92-2f5ffe05e414.png)

Similar issue on PLPs.

Reply:
That's true! I was wondering what would be the best approach for this. It's related to the SEO right?
Maybe we can create a class just for that, maybe we can use the `SROnly` component (because it hides the text from the page, but it's not for this purpose). 🤔

Reply:
That's true. I have just removed the temporary class so far. 
Is this title required? I noticed that on the `Homepage` we don't have it.
And when we add the `Hero` on `PLP` we are going to have an `h1` tag.
Thoughts?

Reply:
I think it was added for SEO purposes.
Hmm. Maybe we could change this title on PDP for an `h1` tag?
![image](https://user-images.githubusercontent.com/3356699/149360829-6f0fb212-6521-43f4-8d8e-f50ef7008492.png)

Not sure; maybe the safest approach now could be: create a new class just for that and keep it.

Reply:
We remove this `h1` tag on PLP since we already have it on the `Hero`. 
On the PDP I agree that we can turn this `h2` into `h1` since we don't have another more "important" option.